src/shared/routes.tsx

```diff
@@ -49,7 +51,8 @@ const routes: RouteWithData[] = [
     loadData: (state, match) =>
       fetchResources(
         match && match.params && (match.params as any)['org'],
-        match && match.params && (match.params as any)['project']
+        match && match.params && (match.params as any)['project'],
+        state && state.nexus && state.nexus.resourcePaginationSettings
```

Comment:
`loadData` is for server-side rendering, so this means the server only use the default pagination state?

Reply:
For the moment, yeah, but when we add url parameterization (if that's the direction we go) this should work to set the correct pagination right out of the box